openapi.yml

```diff
@@ -5523,6 +5683,258 @@ components:
+                            -
+                                field_name: some text
+                                field_value: some text
+                    -
+                        title: some text
+                        sequence: 64
+                        id: 20
+                        page_type: some text
+                        custom_fields:
+                            -
+                                field_name: some text
+                                field_value: some text
+                            -
+                                field_name: some text
+                                field_value: some text
+                invite:
+                    id: 39
+                    first_name: some text
+                    last_name: some text
+                    start_date: '2018-02-10T09:30Z'
+                    location:
+                        id: 21
+                        name: some text
+                    watchlist_colour: GREEN
+                    hosts:
+                        -
+                            id: 16
+                            email: some text
+                            first_name: some text
+                            last_name: some text
+                            profile_pic_url: some text
+                            department: some text
+                            mobile_number: some text
+                        -
+                            id: 92
+                            email: some text
+                            first_name: some text
+                            last_name: some text
+                            profile_pic_url: some text
+                            department: some text
+                            mobile_number: some text
+                    invite_watchlist:
+                        id: 91
+                        external_colours:
+                            - some text
+                            - some text
+                        internal_colours:
+                            - some text
+                            - some text
+                    end_date: '2018-02-10T09:30Z'
+                    email: some text
+                    mobile_number: some text
+                visitor:
+                    id: 20
+                    active: true
+                    company: some text
+                    created_via: some text
+                    email: some text
+                    first_name: some text
+                    last_name: some text
+                    mobile: some text
+                    note: some text
+                    profile_pic_content_type: some text
+                    profile_pic_file_name: some text
+                    profile_pic_file_size: some text
+                    profile_pic_updated_at: some text
+                    watchlist_level: some text
+                    created_at: '2018-02-10T09:30Z'
+                    updated_at: '2018-02-10T09:30Z'
+                    account_id: 82
```

Comment:
This is maybe 🌵 because this is only the example, but you've still got references to `_id` attributes that are not for create/update params